bin/browser-tests

```diff
@@ -11,7 +11,18 @@ my ($cobrand, $coords, $area_id, $name, $mapit_url);
 
 BEGIN {
     $config_file = 'conf/general.yml-example';
-    $cobrand = [ 'borsetshire', 'fixmystreet', 'northamptonshire', 'bathnes', 'buckinghamshire', 'hounslow', 'isleofwight', 'peterborough', 'tfl' ];
+    $cobrand = [qw(
+        borsetshire
+        fixmystreet
+        northamptonshire
+        bathnes
+        buckinghamshire
+        hounslow
+        isleofwight
+        peterborough
+        tfl
+        oxfordshire
```

Comment:
Yes, certainly worth doing this now, but perhaps alphabetise it at the same time? :)